__tests__/cli/deploy/bundle/__snapshots__/DeployBundle.definition.test.ts.snap

```diff
@@ -34,20 +35,21 @@ export const DeployBundleDefinition: ICommandDefinition = {
                  \\"in the target group of CICS regions.\\",
     type: \\"command\\",
     handler: __dirname + \\"/DeployBundle.handler\\",
-    options: [ NameOption, BundledirOption, CicsplexOption, ScopeOption, CsdgroupOption , ResgroupOption, TimeoutOption],
-    profile: { optional: [\\"cics-deploy\\"] },
+    options: [ NameOption, BundledirOption, CicsplexOption, ScopeOption, CsdgroupOption , ResgroupOption, CicshlqOption, TimeoutOption],
+    profile: { required: [\\"zosmf\\"], optional: [\\"cics-deploy\\"] },
     examples: [
         {
             description: \\"Deploy a CICS bundle with a specific name and location to a default set of target regions\\",
-            options: \`--name EXAMPLE --bundledir /u/example/bundleDir --cics-deploy-profile default\`
+            options: \`--name EXAMPLE --bundledir /u/example/bundleDir --cics-deploy-profile default --zosmf-profile zprof\`
```

Comment:
suggest we don't need --zosmf-profile on all of the examples as the user is likely to have a default profile

Reply:
Agreed, I'll change that.